ui/src/components/common/Header.js

```diff
@@ -19,7 +20,9 @@ import { connect } from 'react-redux';
 import { toggleHeaderDropdown } from '../../actions/actions-ui';
 
 class Header extends React.Component {
-  onTitleClick = () => {};
+  onTitleClick () {
+    this.props.history.push('/');
```

Comment:
Are we using this navigation history/ is it wired into React already?

Reply:
It uses Router. the history objection tracks the navigation across all components. It works really well. You can check it by navigating between the home page and the two apps and then using the back button.